_data/toc/graphql.yml

```diff
@@ -138,6 +138,9 @@ pages:
                 - label: updateCartItems mutation
                   url: /graphql/reference/quote-update-cart-items.html
 
+            - label: redeemGiftCardBalanceAsStoreCredit mutation
```

Comment:
Should this be a child of the quote section?

Reply:
No, it's not attached to a cart at all.